packages/ember-views/lib/components/component.js

```diff
@@ -351,6 +351,55 @@ var Component = View.extend(TargetActionSupport, {
     @property hasBlockParams
     @returns Boolean
   */
+
+  /**
+    Enables components to take a list of parameters as arguments
+
+    For example a component that takes two parameters with the names
+    `name` and `age`:
+
+    ```javascript
+    Ember.Component.extend({
+      positionalParams: ['name', 'age']
+    })
+    ```
+
+    It can then be invoked like this:
+
+    ```hbs
+    {{my-component "John" 38}}
+    ```
+
+    The parameters can be refered to just like named parameters:
+
+    ```hbs
+    Name: {{attrs.name}}, Age: {{attrs.age}}.
+    ```
+
+    Using a string instead of an array allows for an arbitrary number of
+    parameters:
+
+    ```javascript
+    Ember.Component.extend({
+      positionalParams: 'names'
+    })
+    ```
+
+    It can then be invoked like this:
+
+    ```hbs
+    {{my-component "John" "Michael" "Scott"}}
+    ```
+
+    The parameters can then be refered to by enumerating over the list:
+
+    ```hbs
+    {{#each attrs.names as |name|}}{{name}}{{/each}}
+    ```
+
+    @public
```

Comment:
I think we also need to add:

@static


This indicates to YUIDoc that it is not an instance property, but a class level static property.